CHANGELOG.md

```diff
@@ -1,5 +1,75 @@
 # Changelog
 
+## [1.27.0] - 2024-01-29
+
+This long-awaited Rustup release has gathered all the new features and fixes since April 2023.
+These changes have greatly improved Rustup's maintainability, user experience, compatibility and documentation quality.
+
+The headlines of this release are:
+- Basic support for `fish` shell has been added.
+- Rustup now uses `clap` v4 for argument parsing.
+- Support and build test for `loongarch64-unknown-linux-gnu` has been added.
+- The project has introduced GitHub Merge Queue as part of its workflow, and the CI pipeline has been revamped accordingly.
+
+Also, the project seems to have attracted a substantial number of new contributors within this release cycle. Looking forward to seeing you again in the future!
```

Comment:
I would make this more factual, like "attracted 15 new contributors this release cycle". (Number to be checked, not sure where you got this?)

It's probably worth mentioning that there are two new team members coordinating this release?

Reply:
> I would make this more factual, like "attracted 15 new contributors this release cycle". (Number to be checked, not sure where you got this?)
GitHub can generate this, and I just checked, this time we have 21 actually!

> It's probably worth mentioning that there are two new team members coordinating this release?
Oh yup. Will do!